src/main/resources/com/unstoppabledomains/config/client/client.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "6.2.1"
+  "version": "6.2.3"
```

Comment:
Any reason it's bumping up to 6.2.3 instead of 6.2.2?

Reply:
I think last time someone did the version bump with git tag and change log but forgot about this file.